gqueries/output_elements/output_series/refinery_189/diesel_distribution_to_industry_in_sankey.gql

```diff
@@ -1,4 +1,8 @@
 # Query for refinery sankey: connection between distribution and use of diesel in the industry sector
 
 - unit = PJ
-- query = DIVIDE(V(industry_final_demand_diesel, demand), BILLIONS)
\ No newline at end of file
+- query = 
+        SUM(
+        DIVIDE(V(industry_final_demand_diesel, demand), BILLIONS),
+        DIVIDE(V(industry_final_demand_diesel_non_energetic, demand), BILLIONS)
+        )
```

Comment:
I would first apply the `DIVIDE`, then the `SUM`. This prevents having to divide twice.